### PR TITLE
Update zkllvm instructions

### LINKED ISSUE #760: Delete zkllvm inputs and use binary proofs as input
- [x] Delete inputs on `zllvm/tests/proofs-of-proofs`
- [x] Modify `bench.sh` to use binary inputs
- [x] Modify README.md and Performance.md to explain this new workflow and point the inputs to the new right location.

### PR DESCRIPTION
**Pull request by yzhang90**
Closes #760

### CODE REVIEW

File: zkllvm/README.md [hunk 1]
```diff
@@ -91,13 +91,10 @@ The rest of the vector is padded by NO-OP instruction till the fixed size is rea
 
 We have a python script translating the binary input:
 ```bash
-python3 translator.py <path-to-assumption> <path-to-claim> <path-to-proof>
+python3 translator.py <path-to-binar-input-dir>
```
Comment (Robertorosmaninho) [inline zkllvm/README.md:94]: `path-to-binary-inputs-dir`

File: Performance.md [hunk 2]
```diff
@@ -153,19 +153,19 @@ binary input format, and therefore, all implementations will share these same
...
-Last Update: Jan 8th, 2024
+Last Update: Jan 22th, 2024
 |       Examples      |CPU Circuit Gen Time | CPU Prove+Verify Time |
 |:-------------------:|:-------------------:|:---------------------:|
-| impreflex           |               9.137 |              1373.576 |
+| impreflex           |               0.585 |               298.686 |
 | transfer-goal       |             147.247 |                     ∞ |
 | batch-transfer-goal |            1407.846 |                     ∞ |
 | perceptron-goal     |             575.171 |                     ∞ |
 | svm-goal            |             585.308 |                     ∞ |
```
Comment (Robertorosmaninho) [inline Performance.md:163]: We should try to generate numbers for the other examples as well! For sure they will be better as well!

**Review by Robertorosmaninho — CHANGES_REQUESTED** [review]

File: zkllvm/README.md (hunk 1 above)
Comment (yzhang90) [reply zkllvm/README.md]: done

File: Performance.md (hunk 2 above)
Comment (yzhang90) [reply Performance.md]: done

**Review by Robertorosmaninho — APPROVED** [review]
Thanks! LGTM!
The gain we get on Circuit generation with it is quite impressive.

**Pull request merged**